warehouse/cli/search/reindex.py

```diff
@@ -68,14 +69,21 @@ def reindex(config, **kwargs):
     random_token = binascii.hexlify(os.urandom(5)).decode("ascii")
     new_index_name = "{}-{}".format(index_base, random_token)
     doc_types = config.registry.get("search.doc_types", set())
+
+    # Create the new index with zero replicas then update it with index
+    # refreshes disabled while we are bulk indexing.
     new_index = get_index(
         new_index_name,
         doc_types,
         using=client,
         shards=config.registry.get("elasticsearch.shards", 1),
-        replicas=config.registry.get("elasticsearch.replicas", 0),
+        replicas=0,
     )
     new_index.create()
+    client.indices.put_settings(
+        index=new_index_name,
+        body={"index": {"refresh_interval": "-1"}}
+    )
```

Comment:
@robhudson Is there any reason why this setting would nee to be set _after_ the index is created?